libraries/AP_HAL_ChibiOS/hwdef/SuccexF4/hwdef.dat

```diff
@@ -0,0 +1,146 @@
+# hw definition file for processing by chibios_pins.py
+# Omnibus F4 PRO, Omnibus F4 V3 boards
+# with F405 mcu, mpu6000 imu, bmp280 barometer, 7456 series osd  and sdcard
+
+MCU STM32F4xx STM32F405xx
+
+HAL_CHIBIOS_ARCH_F405 1
+
+# board ID for firmware load
+APJ_BOARD_ID 143
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+STM32_PLLM_VALUE 8
+
+# board voltage
+STM32_VDD 330U
+
+STM32_ST_USE_TIMER 5
```

Comment:
note that we can now use high timer numbers (12, 13, 14) for system timer, which can make PWM timer assignment easier